polyglot/piranha/src/config.rs

```diff
@@ -42,6 +42,9 @@ pub(crate) struct CommandLineArguments {
   /// Path to output summary json
   #[clap(short = 'j', long)]
   pub(crate) path_to_output_summary: Option<String>,
+  /// Should Piranha apply the replacements to the source files or not
+  #[clap(short = 'd', long, default_value = "false")]
```

Comment:
@ketkarameya I tried to use a boolean type here, but with `default_value_t` set to `false`, it would always return `true`, do you know what is the reason for that behavior?

Reply:
hmm. interesting. Maybe `clap` behaves like Python's `argparse`. 
Anyhoo. 
Can u try - [this](https://github.com/IohannRabeson/lipl/commit/d3073e0eb1f9764c36824122cce55be59527c10f#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR24) ?

Reply:
@ketkarameya thanks, this did work now